### DISCUSSION TITLE
Feedback on Podman Desktop documentation?

### DISCUSSION THREAD

**Discussion by mairin** [podman-desktop/podman-desktop#3319] [Design Feedback]
👋🏻 So I know the category here is on **Design** Feedback, but I wasn't sure what category to put this in. :) We are interested in feedback on our documentation for Podman Desktop, though! This is our current documentation:

https://podman-desktop.io/docs/intro

Note Podman-specific documentation is available here (and that's a bit in flux, we're still converting the docs from the old website format to the new one):

https://podman.io/docs

Some questions I'd like to ask :)

- Have you read our documentation for Podman Desktop before?
- Have you tried to find something in the documentation you couldn't find?
- Is there any kind of tutorial or documentation format you would like to see?
- Do you bounce between the [Podman Desktop documentation](
https://podman-desktop.io/docs/intro) and [Podman documentation](https://podman.io/docs), or tend to stick to Podman Desktop docs? 

Any/all feedback on the documentation is greatly appreciated in this thread :)

**Comment by eidottermihi** [comment]
> Have you read our documentation for Podman Desktop before?

Yes.
>  Have you tried to find something in the documentation you couldn't find?

Yes ("silent install parameter") - but after raising the issue it was included in the docs :)

> Is there any kind of tutorial or documentation format you would like to see?

I would like to see a documentation/tutorial targeted for "system administrators" on how to
- install Podman / Podman Desktop silently
- in an airgapped environment
- including pre-configuring Podman/Podman Desktop
  - to use a corporate proxy
  - to use a corporate registry as search registry and as docker.io mirror (https://podman.io/docs/installation#registriesconf) 
  - to include corporate/internal CA certificates in podman machine (https://github.com/containers/podman/blob/main/docs/tutorials/podman-install-certificate-authority.md)

These topics are all kind of covered in various places - but putting those all together is quite cumbersome.

> Do you bounce between the [Podman Desktop documentation](https://podman-desktop.io/docs/intro) and [Podman documentation](https://podman.io/docs), or tend to stick to Podman Desktop docs?

I keep bouning between Desktop docs and the podman docs (old and new) quite a lot.

**Reply by mairin** [reply]
These are great suggestions, thanks @eidottermihi!

**Comment by rujutashinde** [comment]
closing this discussion as it is either implemented or not needed
